crates/l2/contracts/src/l1/OnChainProposer.sol

```diff
@@ -28,10 +29,21 @@ contract OnChainProposer is IOnChainProposer, ReentrancyGuard {
     /// @dev This is crucial for ensuring that only valid and confirmed blocks are processed in the contract.
     uint256 public lastVerifiedBlock;
 
+    /// @notice The latest committed block number.
+    /// @dev This variable holds the block number of the most recently committed block.
+    /// @dev All blocks with a block number less than or equal to `lastCommittedBlock` are considered committed.
+    /// @dev Blocks with a block number greater than `lastCommittedBlock` have not been committed yet.
+    /// @dev This is crucial for ensuring that only subsequents blocks are committed in the contract.
+    uint256 public lastCommittedBlock;
```

Comment:
We should explain what it means for this variable to be set to `0xFFFFFFFFFFFFFFFF`.

Reply:
Yes, done!